app/queries/get_linkables.rb

```diff
@@ -1,17 +1,37 @@
 module Queries
+  LinkablePresenter = Struct.new(:title, :content_id, :publication_state, :base_path, :internal_name) do
+    def self.from_hash(hash)
+      fields = [
+        hash["title"],
+        hash["content_id"],
+        hash["state"],
+        hash["base_path"],
+        hash["details"].fetch("internal_name", hash['title']),
+      ]
+      new(*fields)
+    end
+  end
+
   class GetLinkables
     def initialize(document_type:)
       @document_type = document_type
     end
 
     def call
-      Linkable
+      documents = ContentItem
         .where(document_type: [document_type, "placeholder_#{document_type}"])
-        .includes(:content_item)
-        .pluck(:content_id, :state, :title, :base_path, "content_items.details->>'internal_name' as internal_name")
-        .map { |linkable_data|
-          Presenters::Queries::LinkablePresenter.present(*linkable_data)
-        }
+        .order('updated_at ASC')
+        .pluck(:updated_at, :content_id)
```

Comment:
It surprises me that this is quicker than `SELECT MAX(updated_at) ...` but it really is. Even with a new index.

```psql
publishing_api_development=> CREATE INDEX idx_temp_dt_ua ON content_items (updated_at, document_type);
CREATE INDEX
Time: 6899.011 ms
publishing_api_development=> EXPLAIN SELECT MAX(updated_at) FROM content_items WHERE document_type = 'placeholder_policy_area';
                                                      QUERY PLAN
-----------------------------------------------------------------------------------------------------------------------
 Result  (cost=20.85..20.86 rows=1 width=0)
   InitPlan 1 (returns $0)
     ->  Limit  (cost=0.42..20.85 rows=1 width=8)
           ->  Index Only Scan Backward using idx_temp_dt_ua on content_items  (cost=0.42..38387.41 rows=1879 width=8)
                 Index Cond: ((updated_at IS NOT NULL) AND (document_type = 'placeholder_policy_area'::text))
(5 rows)
publishing_api_development=> SELECT updated_at, content_id FROM content_items WHERE document_type = 'placeholder_policy_area' ORDER BY updated_at ASC;
Time: 0.801 ms
publishing_api_development=> SELECT MAX(updated_at) FROM content_items WHERE document_type = 'placeholder_policy_area';
            max
----------------------------
 2016-05-03 08:04:58.012402
(1 row)

Time: 14.633 ms
```